src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/PackageSearchResultTablePrinter.cs

```diff
@@ -52,7 +106,7 @@ public void Start()
         /// </summary>
         /// <param name="results">An enumerable of package search metadata to be processed and added to the table.</param>
         /// <param name="table">The table where the results will be added as rows.</param>
-        private static void PopulateTableWithResults(IEnumerable<IPackageSearchMetadata> results, Table table)
+        private async void PopulateTableWithResults(IEnumerable<IPackageSearchMetadata> results, Table table)
```

Comment:
Methods marked async should have the suffix Async:

```suggestion
        private async void PopulateTableWithResultsAsync(IEnumerable<IPackageSearchMetadata> results, Table table)
```